app/pages/api/klima-supply/index.ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiRequest, NextApiResponse } from "next/types";
+import { getKlimaSupply } from "@klimadao/lib/utils";
+
+type ResponseData = {
+    currentKlimaSupply?: string;
+    error?: string;
+};
+
+export default async function handler(
```

Comment:
We can improve the typings a bit:
```suggestion
interface Response {
  currentKlimaSupply: string;
};

const handler: NextApiHandler<Response> = async (req, res) => {
```

Reply:
I realized this API actually needs to be simplified to meet CMC's requirement of just returning the supply and nothing else